sdk/communication/Azure.Communication.CallingServer/tests/CallRecording/ContentDownloadLiveTests.cs

```diff
@@ -0,0 +1,104 @@
+            Assert.AreEqual(documentId, expected.GetProperty("chunkDocumentId").ToString());
+        }
+
+        [Test]
+        public async Task DownloadMetadata_404()
+        {
+            if (SkipCallingServerInteractionLiveTests)
+            {
+                Assert.Ignore("Skip callingserver interaction live tests flag is on");
+            }
+
+            CallAutomationClient client = CreateInstrumentedCallingServerClientWithConnectionString();
+            string documentId = "0-wus-d4-d4c223871b58d3664401b66d35fca785";
+            Uri metadataEndpoint = new Uri($"https://us-storage.asm.skype.com/v1/objects/{documentId}/content/acsmetadata");
+            CallRecording callRecordingClient = client.GetCallRecording();
+            try
+            {
+                Stream metadata = await callRecordingClient.DownloadStreamingAsync(metadataEndpoint);
+                Assert.IsNotNull(metadata);
+                JsonElement expected = JsonDocument.Parse(metadata).RootElement;
+                Assert.AreEqual(documentId, expected.GetProperty("chunkDocumentId").ToString());
+            }
+            catch (RequestFailedException ex)
+            {
+                if (ex.Status == 404)
+                {
+                    Assert.Pass("Recording was not found");
+                }
+                else
+                {
+                    Assert.Fail($"Unexpected error: {ex}");
+                }
+            }
+        }
+
+        [Test]
+        public async Task DownloadMetadataRange()
+        {
+            if (SkipCallingServerInteractionLiveTests)
+            {
+                Assert.Ignore("Skip callingserver interaction live tests flag is on");
+            }
+
+            CallAutomationClient client = CreateInstrumentedCallingServerClientWithConnectionString();
+            int length = 5;
+            string documentId = "0-wus-d4-d4c223871b58d3664401b66d35fca784";
+            Uri metadataEndpoint = new Uri($"https://us-storage.asm.skype.com/v1/objects/{documentId}/content/acsmetadata");
+            CallRecording callRecordingClient = client.GetCallRecording();
+            Response<Stream> response = await callRecordingClient.DownloadStreamingAsync(metadataEndpoint, new HttpRange(0, length));
+            Assert.IsNotNull(response);
+            Assert.AreEqual(length, response.GetRawResponse().Headers.ContentLength);
+        }
+
+        [Test]
+        public async Task DownloadMetadataWithRedirection()
+        {
+            if (SkipCallingServerInteractionLiveTests)
+            {
+                Assert.Ignore("Skip callingserver interaction live tests flag is on");
+            }
+
+            CallAutomationClient client = CreateInstrumentedCallingServerClientWithConnectionString();
+            string documentId = "0-wus-d4-d4c223871b58d3664401b66d35fca784";
+            Uri metadataEndpoint = new Uri($"https://eu-storage.asm.skype.com/v1/objects/{documentId}/content/acsmetadata");
```

Comment:
is the redirection happening in the uri ("eu" instead of "us")? If so, would be better to add a comment for better readability. what do u think?

Reply:
added comment to the test itself.